### ISSUE TITLE
Evaluation Preview Dialog

### ISSUE THREAD

**Issue by ThunderCls** [0xd4d/dnSpy#46]
First of all, let me tell you that this is a great fork of ILSpy, it's just great, though there is something I'm personally missing from ILSpy and that is the instructions (variables and NET controls) evaluation preview dialog, showing attributes and live values while debugging an app. I'm not sure if it's implemented and I don't get it, but this is such a great help for debuggin tasks. Here is a pic of what I mean

![sin titulo](https://cloud.githubusercontent.com/assets/13303069/10744697/e53b53ba-7c09-11e5-9b3f-818c0f381134.png)

With this feature you can get live values/properties/attributes of any local var as well as any control used in the code by just setting the mouse over the line and expanding the treeview or scrolling the listbox, great if dnSpy could get that

**Comment by 0xd4d** [comment]
Try the locals window.

**Comment by ThunderCls** [comment]
Right, the local window, the headache here is that, for example if I wanna see the current value of a window's form control, let's say a TextBox, I should have to go all the way through the endless treeview of locals inside "this" name and locate the control that I'm looking for and then the property. That's really annoying and a waste of time, in my opinion it's so much easy to have this feature added besides the local window like ILSpy,  that's all I meant, anyway if you think that is unnecessary, it's fine, just close this issue.
Thanks

**Comment by Misiu** [comment]
It would be awesome to have this build in. As @ThunderCls wrote before it's waste of time. It is easier to just move mouse over variable and see all it properties.  @0xd4d please consider adding this in future version.

**Comment by eng-marani** [comment]
Local window is very unfriendly from usability point of view. On real user cases finding the wanted variable is a nightmare. The bare minimum for an usable UI, in my humble opinion, should include:

1- Filter box, user should be able to limit display to only filtered variables by name or value (this is a different concept from search box)

2- Sortable listview clicking on captions

3- Pinpoint of some key locals in a specific window, like visual studio does, with an "add watchdog" RMB on live code vars, if not a flyout inspector if difficult to implement
